lynx/common/actions/move.py

```diff
@@ -1,17 +1,18 @@
 from dataclasses import dataclass
```

Comment:
> Imports should be grouped in the following order:
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.

https://peps.python.org/pep-0008/#imports
Just do that in the files that you have changed. I will do the rest later.